tests/test_C0_data_processing.py

```diff
@@ -268,28 +286,66 @@ def test_define_energyBusses():
-        ), f"Bus {k+BUS_SUFFIX} of the input busses is not added to the {ENERGY_BUSSES}."
-    for k in out_bus_names:
-        assert (
-            k + BUS_SUFFIX in dict_test[ENERGY_BUSSES].keys()
-        ), f"Bus {k+BUS_SUFFIX} of the output busses is not added to the {ENERGY_BUSSES}."
+        asset_names[0] in dict_test[ENERGY_BUSSES][in_bus_names[0]][ASSET_DICT]
+    ), f"Asset {asset_names[0]} not added to the connected bus {in_bus_names[0]}"
+    assert (
+        asset_names[1] in dict_test[ENERGY_BUSSES][in_bus_names[1]][ASSET_DICT]
+    ), f"Asset {asset_names[1]} not added to the connected bus {in_bus_names[1]}"
+    assert (
+        asset_names[2] in dict_test[ENERGY_BUSSES][in_bus_names[2]][ASSET_DICT]
+    ), f"Asset {asset_names[2]} not added to the connected bus {in_bus_names[2]}"
+    assert (
+        asset_names[4] in dict_test[ENERGY_BUSSES][in_bus_names[3]][ASSET_DICT]
+    ), f"Asset {asset_names[4]} not added to the connected bus {in_bus_names[3]}"
+    assert (
+        asset_names[5] in dict_test[ENERGY_BUSSES][in_bus_names[4]][ASSET_DICT]
+    ), f"Asset {asset_names[5]} not added to the connected bus {in_bus_names[4]}"
+    assert (
+        asset_names[5] in dict_test[ENERGY_BUSSES][in_bus_names[5]][ASSET_DICT]
+    ), f"Asset {asset_names[5]} not added to the connected bus {in_bus_names[5]}"
+    assert (
+        asset_names[0] in dict_test[ENERGY_BUSSES][out_bus_names[0]][ASSET_DICT]
+    ), f"Asset {asset_names[0]} not added to the connected bus {out_bus_names[0]}"
+    assert (
+        asset_names[1] in dict_test[ENERGY_BUSSES][out_bus_names[1]][ASSET_DICT]
+    ), f"Asset {asset_names[1]} not added to the connected bus {out_bus_names[1]}"
+    assert (
+        asset_names[3] in dict_test[ENERGY_BUSSES][out_bus_names[2]][ASSET_DICT]
+    ), f"Asset {asset_names[3]} not added to the connected bus {out_bus_names[2]}"
+    assert (
+        asset_names[4] in dict_test[ENERGY_BUSSES][out_bus_names[3]][ASSET_DICT]
+    ), f"Asset {asset_names[4]} not added to the connected bus {out_bus_names[3]}"
+    assert (
+        asset_names[5] in dict_test[ENERGY_BUSSES][out_bus_names[4]][ASSET_DICT]
+    ), f"Asset {asset_names[5]} not added to the connected bus {out_bus_names[4]}"
+    assert (
+        asset_names[5] in dict_test[ENERGY_BUSSES][out_bus_names[5]][ASSET_DICT]
+    ), f"Asset {asset_names[5]} not added to the connected bus {out_bus_names[5]}"
 
 
-def test_add_busses_of_asset_depending_on_in_out_direction_single():
+def test_add_asset_to_asset_dict_for_each_flow_direction():
     bus_names = ["bus_name_" + str(i) for i in range(1, 3)]
     asset_name = "asset"
     asset_label = "asset_label"
     energy_vector = "Electricity"
     dict_test = {
-        ENERGY_BUSSES: {},
+        ENERGY_BUSSES: {
+            bus_names[0]: {LABEL: bus_names[0], ENERGY_VECTOR: energy_vector},
+            bus_names[1]: {LABEL: bus_names[1], ENERGY_VECTOR: energy_vector},
```

Comment:
```suggestion
            bus_name: {LABEL: bus_name, ENERGY_VECTOR: energy_vector}
            for bus_name in bus_names
```

Reply:
Bit this is not always the case. Sometimes its asset 4 and bus 5, for example

Reply:
ah ok